services/state/token/account.proto

```diff
@@ -30,167 +30,165 @@ option java_package = "com.hederahashgraph.api.proto.java";
 option java_multiple_files = true;
 
 /**
- * First-draft representation of a Hedera Token Service account entity in the network Merkle tree.
+ * Representation of a Hedera Token Service account entity in the network Merkle tree.
  *
- * As with all network entities, account has an unique entity number represented as shard.realm.X.
+ * As with all network entities, account has a unique entity number represented as shard.realm.X.
  * X can be an alias public key or an EVM address or a number.
- *
  */
 
 message Account {
     /**
      * The unique entity id of the account.
      */
-    AccountID account_id = 1;
+    AccountID account_id = 25;
     /**
      * The alias to use for this account, if any.
      */
-    bytes alias = 2;
+    bytes alias = 23;
     /**
      * (Optional) The key to be used to sign transactions from the account, if any.
      * This key will not be set for hollow accounts until the account is finalized.
      * This key should be set on all the accounts, except for immutable accounts (0.0.800 and 0.0.801).
      */
-    Key key = 3;
+    Key key = 26;
     /**
      * The expiration time of the account, in seconds since the epoch.
      */
-    int64 expiry = 4;
+    int64 expiration_seconds = 12;
```

Comment:
I believe we agreed to use singular for point-in-time and plural only for durations?

Reply:
Fixed.